services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/rabbitmq.py

```diff
@@ -168,8 +174,10 @@ async def post_log_message(self, log_msg: Union[str, List[str]]) -> None:
         if isinstance(log_msg, str):
             log_msg = [log_msg]
 
+        time_string = _get_utc_now().strftime(LOG_DATETIME_FORMAT)
```

Comment:
so if I get it correctly you actually add the timestamp when logging to rabbitMQ right?
in the dask-sidecar I already get the timestamps from the docker container directly, see [here](https://github.com/ITISFoundation/osparc-simcore/blob/247620dae25c67169e43483fe9a0cbdc84ad31d6/services/dask-sidecar/src/simcore_service_dask_sidecar/computational_sidecar/docker_utils.py) if possible which I think you can.
I am missing a timestamp for messages not from the container, that is right, but I can see that if we need to debug it is probably easier if the timestamp is the same as what we would see in graylog.

Shall we go for adding a timestamp only if it is missing? or having a rabbitMQ timestamp and a container timestamp? I mean the message in rabbitMQ is pretty flexible. I think we can pass a json with several fields. what do you think?